core/src/nozzle_subgraph/runner/compat.rs

```diff
@@ -0,0 +1,50 @@
+use alloy::primitives::{BlockHash, BlockNumber};
+use chrono::{DateTime, Utc};
+
+mod legacy {
+    pub(super) use graph::{
+        blockchain::{BlockHash, BlockPtr, BlockTime},
+        components::store::BlockNumber,
+        data::store::scalar::Timestamp,
+    };
+}
+
+pub(super) trait Compat<T> {
```

Comment:
Not a big deal, but `Compat` is maybe a bit of a misnomer, maybe `FromAlloy` would be better